cmd/cosign/cli/verify/verify.go

```diff
@@ -261,14 +288,26 @@ func (c *VerifyCommand) Exec(ctx context.Context, images []string) (err error) {
 			if err != nil {
 				return err
 			}
+		case co.RootCerts != nil:
+			// Verify certificate with root (and if given, intermediate) certificate
+			pubKey, err = cosign.ValidateAndUnpackCert(cert, co)
+			if err != nil {
+				return err
+			}
+		default:
+			return errors.New("internal error in handling CertChain and RootCerts - default case should never happen")
```

Comment:
Can the error text be something about "no certificate chain provided to verify certificate"? Even if it's never going to happen, that error text seems more accurate.